lte/gateway/c/core/oai/tasks/s6a/s6a_messages.hpp

```diff
@@ -25,33 +25,30 @@
 #define S6A_MESSAGES_H_
 
 #include "lte/gateway/c/core/common/common_defs.h"
-#include "lte/gateway/c/core/oai/include/s6a_messages_types.h"
+#include "lte/gateway/c/core/oai/include/s6a_messages_types.hpp"
```

Comment:
Include the header files as mentioned in https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes